EzMap.Domain/Repositories/PoiRepository.cs

```diff
@@ -9,45 +9,49 @@ public interface IPoiRepository
 {
     void AddPoi(PoiCreateDto dto);
 
-    Task ReadPoisAsync(CancellationToken token = default);
+    Task<List<Poi>> GetListPoiAsync(CancellationToken token = default);
```

Comment:
Why shouldn't `UpdatePoiAsync` and `DeletePoiAsync` use CancellationToken as well?